Makefile

```diff
@@ -35,14 +31,3 @@ imports: ${GO_SRCS}
 	@echo "Checking go imorts."
 	@echo "goimports -l . | \grep -v vendor"
 	@[ -z "$(shell goimports -l . | \grep -v vendor | \tee /dev/stderr)" ]
-
-.PHONY: clean
-clean:
-	rm -f resources/bindata.go
-
-# generate & format bindata.go.
-resources/bindata.go: $(shell find resources -type f -name '*.json')
-	rm -f $@
-	go-bindata -o resources/bindata.go -pkg resources resources/
-	goimports -w resources/bindata.go
```

Comment:
I think we don't have to do `goimports` and `gofmt` for the generated `bindata.go` file.